docs/manual/kinds/vr-openbsd.md

```diff
@@ -0,0 +1,70 @@
+---
+search:
+  boost: 4
+---
+# OpenBSD
+
+[OpenBSD](https://www.openbsd.org/) is identified with `vr-openbsd` kind in the [topology file](../topo-def-file.md). It is built using [vrnetlab](../vrnetlab.md) project and essentially is a Qemu VM packaged in a docker container format.
+
```

Comment:
@dteslya thanks!

Would you mind adding a chapter 
```
## Getting OpenBSD image
```
where you explain where to get the image with cloud-init?

Reply:
Done.